docs/topics/Permissions.md

```diff
@@ -33,15 +33,15 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | BAN_MEMBERS \*                | `0x0000000000000004` `(1 << 2)`  | Allows banning members                                                                                                                              |              |
 | ADMINISTRATOR \*              | `0x0000000000000008` `(1 << 3)`  | Allows all permissions and bypasses channel permission overwrites                                                                                   |              |
 | MANAGE_CHANNELS \*            | `0x0000000000000010` `(1 << 4)`  | Allows management and editing of channels                                                                                                           | T, V, S      |
-| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management and editing of the guild                                                                                                          |              |
+| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management of the server name, region, and adding bots to the server                                                                          |              |
```

Comment:
```suggestion
| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows deleting invites, editing guild settings and welcome screen and managing guild template, integrations and widget                             |              |
```

Reply:
"editing guild settings" *includes* welc screen, template, widget and integrations, i think its just easier to leave it as Allows management and editing of the guild

Reply:
Editing [guild settings](https://discord.com/developers/docs/resources/guild#modify-guild) and [welcome screen](https://discord.com/developers/docs/resources/guild#modify-guild-welcome-screen) and managing [guild template](https://discord.com/developers/docs/resources/guild-template#modify-guild-template), [integrations](https://discord.com/developers/docs/resources/guild#delete-guild-integration) and [widget](https://discord.com/developers/docs/resources/guild#modify-guild-widget) are all different things in the term of API, they use clearly *different* endpoints, and my suggestion reflects them, assuming OP's intention is to clarify things.

Reply:
> Editing [guild settings](https://discord.com/developers/docs/resources/guild#modify-guild) and [welcome screen](https://discord.com/developers/docs/resources/guild#modify-guild-welcome-screen) and managing [guild template](https://discord.com/developers/docs/resources/guild-template#modify-guild-template), [integrations](https://discord.com/developers/docs/resources/guild#delete-guild-integration) and [widget](https://discord.com/developers/docs/resources/guild#modify-guild-widget) are all different things in the term of API, they use clearly _different_ endpoints, and my suggestion reflects them, assuming OP's intention is to clarify things.

Is there a way to insert those links into the docs so people can see those descriptions of what each of those things entails?